modules/custom/az_core/az_core.module

```diff
@@ -529,3 +530,18 @@ function az_core_preprocess_container__text_format_filter_help(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_entity_embed_container().
+ *
+ * Prevents competing submit button lockup errors in edit pages
+ * by swapping embedded webforms with a custom preview when in collapsed mode.
+ */
+function az_core_preprocess_entity_embed_container(array &$variables) {
+  if (isset($variables['element']['entity']['webform'])) {
+    $embedded_webform_preview = $variables['element']['entity']['title'][0]['#context']['value'];
+    if (\Drupal::routeMatch()->getRouteName() === 'entity.node.edit_form') {
+      $variables["children"] = Markup::create('<hr><p><h3>' . $embedded_webform_preview . ' (Preview)</h3></p><hr>');
```

Comment:
```suggestion
      // Get the view builder for the entity.
      $entity = $variables["element"]['#entity'];
      $type_id = $entity->getEntityTypeId();
      $builder = \Drupal::entityTypeManager()->getViewBuilder($type_id);
      // Render the entity as a teaser.
      $variables['children'] = $builder->view($entity, 'teaser');
```
One option would be rendering the entity as a teaser instead of the full view for the edit form.